include/seqan3/search/fm_index/bi_fm_index.hpp

```diff
@@ -72,7 +72,12 @@ class bi_fm_index
     using sdsl_index_type = sdsl_index_type_;
 
     //!\brief The type of the underlying SDSL index for the reversed text.
-    using rev_sdsl_index_type = sdsl_index_type_;
+    using rev_sdsl_index_type = sdsl::csa_wt<sdsl_wt_index_type::wavelet_tree_type, // Wavelet tree type
+                                             10'000'000, // Sampling rate of the suffix array
+                                             10'000'000, // Sampling rate of the inverse suffix array
```

Comment:
Just a side question. Why do we need an inverse suffix array?

Reply:
Not too sure, AFAIK it's sometimes used for construction stuff; but we don't need it - but SDSL want's a rate anyway.

Reply:
I hope that it's not build if it\s not needed :see_no_evil: